website/content/docs/commands/policies/index.mdx

```diff
@@ -1,8 +1,8 @@
 ---
 layout: docs
 page_title: policies - Command
-description: |-
-  The "policies" command lets you perform operations on Boundary policy resources.
+description: >-
+  The "policies" command performs operations on policy resources. Policies codify how long Boundary retains session recordings.
```

Comment:
just noticing an inconsistency in adding the work Boundary to other descriptions. I'll add it when I notice it 

```suggestion
  The "policies" command performs operations on Boundary policy resources. Policies codify how long Boundary retains session recordings.
```